test/test_fcntl.rs

```diff
@@ -30,6 +30,9 @@ use std::os::unix::fs;
 
 #[test]
 #[cfg(not(target_os = "redox"))]
+// QEMU does not handle openat well enough to satisfy this test
```

Comment:
What's the status of running the tests locally for you?

Reply:
The tests all pass for me under qemu-arm, having ignored `test_openat` and the two OFD flock tests.  (The OFD flock fcntls are not supported on uclibc.)

I have run a subset of the tests on a physical device, but I don't have a device which runs a new enough uclibc & kernel to run every test.  Those which can run have passed.

Reply:
Oh, and the `test_openat` test does succeed on actual hardware.  :+1:

Reply:
Wow, that's some surprisingly basic functionality to be broken in user-mode emulation.